seed/decorators.py

```diff
@@ -99,8 +99,12 @@ def wrapper(request, *args, **kwargs):
         response = func(request, *args, **kwargs)
 
         status_code = 200
-        if response.get('status') == 'error' or response.get('success') == False:
-            status_code = 400
+
+        # This is somewhat hackish, but the response is not always an HttpResponse. I'm explicitly
+        # excluding the list because it is a known type that is returned in the cleansing routines.
+        if not isinstance(response, list):
```

Comment:
Would it make more sense to test if response is an HttpResponse before calling get(), as opposed to testing if it's not a list?